proto/errorpb.proto

```diff
@@ -23,7 +23,7 @@ message NotLeader {
 
 message DiskFull {
     // The requested store ID
-    uint64 store_id = 1;
+    repeated uint64 store_id = 1;
```

Comment:
why should we use repeated?

Reply:
The Region leader can be ok but majority peers are in disk full states. This case can be detected by the leader and a DiskFull error will be returned.